docker/build.sh

```diff
@@ -39,7 +39,7 @@ echo "docker system prune -f -a"
 
 #cmssw_pkgs="proxy frontend exporters exitcodes nats-sub das dbs2go dbs couchdb reqmgr2 reqmgr2ms reqmon workqueue acdcserver confdb crabserver crabcache cmsmon dmwmmon dqmgui t0_reqmon t0wmadatasvc dbsmigration phedex httpgo httpsgo"
 
-cmssw_pkgs="proxy frontend exporters exitcodes nats-sub das das-server das-mongo dbs2go dbs reqmgr2 reqmgr2ms reqmon workqueue confdb crabserver crabcache cmsmon dmwmmon t0_reqmon t0wmadatasvc dbsmigration httpgo httpsgo"
+cmssw_pkgs="reqmon workqueue confdb crabserver crabcache cmsmon dmwmmon t0_reqmon t0wmadatasvc dbsmigration httpgo httpsgo"
```

Comment:
Why did you comment out cmssw_pkgs like proxy, das-server, etc.?

Reply:
Oh. It was by mistake. I have removed build script from this PR now.